### DISCUSSION TITLE
Cant create a foreign key array

### DISCUSSION THREAD

**Discussion by R-Mull** [supabase/supabase#19961] [Questions] [answered]
I'm very new to Supabase and not a developer so I'm slowly working my way through.
When creating my tables, I keep getting errors when I try to create a foreign key array.

I have a table for 'companies' and another table called 'company_sites', I want to create a column called 'sites' in the table 'company' that shows a list of sites that belong to that company.

I keep getting the same error and don't know how to fix it.
I have the same issue when I try creating a list of staff at the company and reference another table.

<img width="564" alt="Fkey array error" src="https://github.com/supabase/supabase/assets/154046331/1e00125a-6e2d-4dca-a4ff-7745fef58517">

Any help on this would be greatly appreciated.
Cheers

**Comment by GaryAustin1** [comment, marked as answer]
That is not possible in Postgres.  You would need to create a join table with foreign keys to both tables and the populate pairs of keys for each. 
https://supabase.com/docs/guides/database/joins-and-nesting#many-to-many-joins
https://supabase.com/docs/guides/database/tables#joining-tables-with-foreign-keys

**Reply by R-Mull** [reply]
Thanks for replying @GaryAustin1, I'm coming from using Xano so I assumed this would be a standard feature in Supabase too.
